packages/webapp/src/components/Map/Modals/BulkSensorUploadModal/useValidateBulkSensorData.js

```diff
@@ -169,21 +212,20 @@ export function useValidateBulkSensorData(onUpload, t) {
     }
   };
 
-  const generateADownload = (s) => {
-    var buf = new ArrayBuffer(s.length);
-    var view = new Uint8Array(buf);
-    for (var i = 0; i != s.length; ++i) view[i] = s.charCodeAt(i) & 0xff;
-    return buf;
-  };
+  const generateErrorFormat = (errors) =>
```

Comment:
Can we make this a utility function? In `packages/webapp/src/util`, so that it can be accessed in other places in the app (ex. sensor upload fail notification).

Reply:
Also, can you make a comment for what the input type is?

Reply:
Done